Automattic/jetpack
# Payment blocks: Reset to default values after product creation

Fixes # (https://github.com/Automattic/jetpack/issues/23575)

#### Changes proposed in this Pull Request:
In Payment button block after creation of new subscription reset form data to default values.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Test on paid account with stripe connection
* Create new post
* Add block "Payment Button"
* Add new payment plan
* Change all form data like title, currency, amount...
* Click "add payment play"
* Expected behaviour:  All fields should be reset to default values

<img width="900" alt="payment-button" src="https://user-images.githubusercontent.com/82798599/165929440-4d8e1c85-bd92-440c-9922-3b654dd529d2.png">
